### ISSUE TITLE
[Done] Explanation Regarding Node Configuration

### ISSUE THREAD

**Issue by junkangwu** [Open-Reasoner-Zero/Open-Reasoner-Zero#5]
Thank you very much for your work and your report; I found it very insightful. I have two questions about `Start Orz-7B PPO Training`:

**Question 1:**  
In the line `ref_num_nodes: int = total_num_nodes = 32`, does `total_num_nodes` mean that 32 nodes are required? However, when I checked while debugging, this value is 8? So, should this represent the number of GPU cards instead?

**Question 2:**  
If I have 4 servers, each with 8 A100 GPUs, how should I modify the parameters to run the code?

Thank you!

**Comment by REIGN12** [comment]
Many thanks for your attention!
[This line](https://github.com/Open-Reasoner-Zero/Open-Reasoner-Zero/blob/main/playground/orz_7b_ppo.py#L95) means for orz_ppo_7b experiments, in total 32 gpus are needed. 

Note that we have colocate policy, critic, ref_model [in this line to control this](https://github.com/Open-Reasoner-Zero/Open-Reasoner-Zero/blob/main/playground/orz_7b_ppo.py#L104), so all of them share the same 32 gpus

For DEBUG_MODE=True, all of them are colocated on 8 gpus, which means you can debug on 1x8H800 GPUs.

**Comment by REIGN12** [comment]
If you have 4 servers, you can just run orz_ppo_7b with DEBUG_MODE=False and check for the multi node starting command in the README, without any change on the python code.

**Comment by junkangwu** [comment]
Thank you for your response; I understand how to handle the training nodes.

I have one more question regarding the original text:  
Figures 14 and 15 illustrate the effects of modifying KL Loss and sampling strategies, respectively.

1. Why is the response length decreasing across the board, even for the recommended settings of lambda=1.0, KL=0.0, and T=1.0? Is this due to results from continuing training from a specific checkpoint?
2. Why is the response length already quite large (greater than 1k) at step 0?

**Comment by REIGN12** [comment]
Many thanks for your careful inspections!
Due to limited time, we have conducted some ablation studies on other settings(different from the final setting described in the main content).

Regarding to Fig 14 and 15, these experiments are conducted with QwenMath-7B model as initialization and use math train set as training data, and with different hyperparameters, thus the details are not directly comparable. 

Many thanks for your comments again! We are actively organizing our experiments and update more results as soon as possible. Please stay tuned!
